content/en-us/projects/managing.md

```diff
@@ -229,49 +227,6 @@ Showcase](/resources/beyond-the-dark) and [The Mystery of Duvall Drive](../resou
 
 <img src="../assets/resources/beyond-the-dark/building-architecture/Architecture-Overview.jpeg" />
 
-## Optimization and Performance
```

Comment:
All of this is outdated and has been replaced by the new, dedicated "Performance Optimization" articles also in the Projects category.